sentry/src/main/java/io/sentry/util/TracingUtils.java

```diff
@@ -116,4 +117,32 @@ public TracingHeaders(
       return baggageHeader;
     }
   }
+
+  /** Checks if a transaction has been ignored. */
```

Comment:
(l) Not 100% sure about the comment here. Maybe something like this?
```suggestion
  /** Checks if a transaction is to be ignored. */
```